public/js/projectView.js

```diff
@@ -11,5 +15,11 @@ Project.makeSectionsAppear = function() {
 
 
 $(document).ready(function() {
```

Comment:
You should try and refrain from using a `$(document).ready` and instead invoke a function at the bottom of the HTML page, just like in the lab.

Reply:
fixed!